.gitignore

```diff
@@ -15,3 +15,14 @@ build/
 # IntelliJ
 *.iml
 .idea/
+acmepos/.settings/
```

Comment:
All of these are redundant, just include `.classpath/`, `.settings/`, `.project` and `bin/`.

Reply:
fixed